lib/browser/api/browser-window.ts

```diff
@@ -199,7 +199,14 @@ BrowserWindow.prototype.setBackgroundThrottling = function (allowed: boolean) {
 };
 
 BrowserWindow.prototype.addBrowserView = function (browserView: BrowserView) {
-  if (browserView.ownerWindow) { browserView.ownerWindow.removeBrowserView(browserView); }
+  if (this._browserViews.includes(browserView)) {
+    return;
+  }
```

Comment:
This introduces a breaking change to the z-order of `BrowserView`s: Previously, calling `addBrowserView` would make the view the top-most view.

This is so useful that it's [documented for the successor API](https://www.electronjs.org/docs/latest/api/view#viewaddchildviewview-index).

Reply:
@nikwen strictly speaking we broke it without documenting a change so it's not a breaking change in this case - i'm fine with potentially breaking it but it should go through a cycle then and we've deprecated BrowserView already so there's not really a point imo

Reply:
I don't have time to look into it further right now. If Shelley thinks it's fine, I'm okay with it.